src/ZODB/blob.py

```diff
@@ -733,7 +733,6 @@ def temporaryDirectory(self):
         return self.fshelper.temp_dir
 
 
-@zope.interface.implementer(ZODB.interfaces.IBlobStorage)
```

Comment:
I didn't encounter this as being inconsistent when I was looking at https://github.com/zopefoundation/zope.interface/issues/207. Did you find issues with it? 

I tend to like the static definition, even though the class itself can't be validated as such. What if the `directlyProvides` block in `__init__` was only executed in the `IBlobStorageRestorable` case? (Doesn't that interface extend `IBlobStorage`?)

Not super important though.

Reply:
> Did you find issues with it?

I dove into the ZODB package because [zope.session](https://github.com/zopefoundation/zope.session/pull/13) the strict iro checker complains about ZODB when I run the zope.session tests with strict iro. 

I like your suggestion of [not touching the static definition](14e73eb3d6c273fd3227885de9d83009ba3ebb02).